src/plugins/discover/public/application/components/doc/use_es_doc_search.ts

```diff
@@ -30,20 +30,24 @@ export function buildSearchBody(
   useNewFieldsApi: boolean
 ): Record<string, any> {
   const computedFields = indexPattern.getComputedFields();
-  const runtimeFields = indexPattern.getComputedFields().runtimeFields;
-  return {
+  const runtimeFields = computedFields.runtimeFields;
+  const request: Record<string, any> = {
```

Comment:
We have proper typing for ES requests in `import type { estypes } from '@elastic/elasticsearch';` to avoid introducing any occurance of `any` here.